README.rst

```diff
@@ -211,6 +222,8 @@ The C++ code can be tested with
 
 .. installation_LGPU-start-inclusion-marker-do-not-remove
 
+--------------------------
```

Comment:
Should we also add this above Lightning Qubit's installation section?

Reply:
Ok I will add it